WordPress/src/main/res/layout/activity_log_list_event_item.xml

```diff
@@ -17,6 +17,7 @@
         android:layout_centerVertical="true"
         android:layout_margin="@dimen/activity_log_icon_margin"
         android:contentDescription="@string/activity_log_icon"
```

Comment:
Hi @A-L-Fearon what do you think about removing the `contentDescription` since setting `importantForAccessibility` to `no` makes it redundant?

Reply:
Heya @jd-alexander it does indeed make it redundant, thanks for pointing it out. I'll make the change.

Reply:
Thanks!